README.md

```diff
@@ -1,16 +1,56 @@
 # TOKBOX INTEGRATION
 
-The following files are in this repo for a code review to find possible ways to improve the use of the Tokbox JS SDK, the error handling, the general performance, etc.
+# Code Review
 
-- TokboxWebsocket
-- TokboxSession
-- TokboxSessionHelpers
-- TokboxPublisher
-- TokboxScreenPublisher
-- TokboxSubscriber
-- TokboxStreamStats
+The following files are in this repo for a code review to find possible ways to improve how we use the Tokbox JS SDK. Some points for the review are the error handling, the optimization and the monitoring of the streaming quality and the general performance.
 
-We use these integration in Demio Viewer, here join links for testing two presenters:
+- **TokboxWebsocket.js**
 
-- https://event.staging.demio.com/join/[demo-hash]
-- https://event.staging.demio.com/join/[demo-hash]
\ No newline at end of file
+  Handles the websocket connection to our singaling server. We use Tokbox NODE SDK in that server to create the sessions, user tokens, start archiving... 
+
+- **TokboxSession.js**
+
+  Handles the Session initilization, connection and attaches the session event listeners after the Tokbox webscket receives the Session ID, apiKey and user token.
+
+- **TokboxSessionHelpers.js**
+
+  Contains the event listener callbacks for the Session, Connection and Streams. Also, shares functions for error handling and UI helpers.
+
+- **TokboxPublisher.js**
+
+  Handles the publisher initialization, publication of audio and video streams, change of the audio or video input. Handles errors and attaches the publisher event listeners. Subcribes to its own stream for monitoring the stats about the streaming quality. Uses our hack for the audio autoplay.
```

Comment:
What do you mean by "Subcribes to its own stream for monitoring the stats about the streaming quality"?

Reply:
When a publisher is created a subscriber is created in the same browser using networkTest:true to test the download bitrate.